receiver/oracledbreceiver/metadata.yaml

```
@@ -109,6 +109,51 @@ metrics:
       value_type: int
       input_type: string
     unit: "{reads}"
+  oracledb.physical_reads_direct:
+    description: Number of reads directly from disk, bypassing the buffer cache
+    enabled: false
+    sum:
+      aggregation_temporality: cumulative
+      monotonic: true
+      value_type: int
+      input_type: string
+    unit: "{reads}"
+  oracledb.physical_read_io_requests:
+    description: Number of read requests for application activity
+    enabled: false
+    sum:
+      aggregation_temporality: cumulative
+      monotonic: true
+      value_type: int
+      input_type: string
+    unit: "{requests}"
+  oracledb.physical_writes:
+    description: Number of physical writes
+    enabled: false
+    sum:
+      aggregation_temporality: cumulative
+      monotonic: true
+      value_type: int
+      input_type: string
+    unit: "{writes}"
+  oracledb.physical_writes_direct:
+    description: Number of writes directly from disk, bypassing the buffer cache
```

```suggestion
    description: Number of writes directly to disk, bypassing the buffer cache
```
I _think_ this should be it?

---

Yes, you are correct, good catch!